app/models/document.rb

```diff
@@ -239,7 +239,7 @@ def publish!
         email_alert_api.send_alert(EmailAlertPresenter.new(self).to_json)
       end
 
-      publish_request.code == 200 && rummager_request.code == 200
+      publish_request.code == 200 && rummager_request.code == 202
```

Comment:
When we originally prototyped the phase 2 stuff @elliotcm did some work on async notifications, but we haven't had time to take it any further. It's definitely in the long-term roadmap; happy to have a discussion to thresh out how it should work.